package.json

```diff
@@ -54,8 +54,9 @@
     "js-beautify": "^1.9.1",
     "lodash.merge": "^4.6.1",
     "mkdirp": "^0.5.1",
-    "mocha": "^4.1.0",
-    "mocha-junit-reporter": "^1.22.0",
+    "mocha": "6",
```

Comment:
maybe `"mocha": "^6"` ?

Reply:
Or try the latest 
```suggestion
    "mocha": "^6.2.0",
```

Reply:
techincally, there are no difference between
`6` and `^6` 
`^6.2.0` will not install all under 6.2.0

I don't think that versions equal and higher than 6.2.0 will be removed from package servers.

Minor and patch versions updates are not include breaking changes, so it will not broken.
Only new non-breaking features and bug fixed.

But ok, I'll change to today last `^6.2.0`

Reply:
And Codeceptjs does not use lock files.